Makefile

```diff
@@ -263,7 +263,7 @@ space := $() $()
 comma := ,
 
 # Define the compiler Cargo features for all crates.
-compiler_features := --features $(subst $(space),$(comma),$(compilers)),wasmer-artifact-create,static-artifact-create,wasmer-artifact-load,static-artifact-load
+compiler_features := --features $(subst $(space),$(comma),$(compilers)),wasmer-artifact-create,static-artifact-create,wasmer-artifact-load,static-artifact-load,experimental-io-devices
```

Comment:
Can we remove this from the default compiler features? Turning on exprimental io devices added some dependencies on other systems that were not ideal for CLI usage